src/plugins/index_pattern_management/public/components/create_index_pattern_wizard/types.ts

```diff
@@ -92,4 +92,5 @@ export interface DataSourceTableItem {
   sort: string;
   checked?: 'on' | 'off';
   label: string;
+  datasourceversion: string;
```

Comment:
Is it expected that each plugin needs to have this change in order to onboard version decoupling?

Reply:
Yes, datasource would be the minimal fields to check in order to decide the version compatibility